openedx/core/djangoapps/persistenttask/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Thoughts on renaming this folder to something like "celery_utils"?  The current name also no longer matches the name of the class, `PersistOnFailureTask`.  Or, we can also rename the folder (and add subfolders) as we add more celery task decorator utilities.